code/__DEFINES/spaceman_dmm.dm

```diff
@@ -0,0 +1,13 @@
+// Interfaces for the SpacemanDMM linter, define'd to nothing when the linter
+// is not in use.
+
+// The SPACEMAN_DMM define is set by the linter and other tooling when it runs.
+#ifdef SPACEMAN_DMM
+	#define RETURN_TYPE(X) set SpacemanDMM_return_type = X
+	#define SHOULD_CALL_PARENT(X) set SpacemanDMM_should_call_parent = X
+	#define UNLINT(X) SpacemanDMM_unlint(X)
```

Comment:
What's the purpose of this one ?

Reply:
Yeah, it's used in the `WARNING` macro to suppress lints about the use of `src` in global `/proc`s where `src` is guaranteed to be null.